chain/index/gc.go

```diff
@@ -75,17 +76,24 @@ func (si *SqliteIndexer) gc(ctx context.Context) {
 	// -------------------------------------------------------------------------------------------------
 	// Also GC eth hashes
 
-	// Convert gcRetentionEpochs to number of days
-	gcRetentionDays := si.gcRetentionEpochs / (builtin.EpochsInDay)
-	if gcRetentionDays < 1 {
-		log.Infof("skipping gc of eth hashes as retention days is less than 1")
-		return
-	}
+	// Calculate the retention duration based on the number of epochs to retain.
+	// retentionDuration represents the total duration (in nano seconds) for which data should be retained before considering it for garbage collection.
```

Comment:
how is it in nanoseconds ?

Reply:
It will be seconds 👍🏾 (I will update it)
so what is happening here is we are converting EpochDurationSeconds(30) to its time duration by multiplying with time.seconds.

Reply:
So `time.Duration` takes data in `nanoseconds`, 
so if I just multiply `time.Duration(rententionEpoch * EpochsDurationSeconds)`, it will give 600 (rententionEpoch = 20, EpochsDurationSeconds=30) but that will be in **nanoseconds**, to get proper time in seconds we have to multiply with `time.Seconds`